api/pom.xml

```diff
@@ -331,7 +331,10 @@
                         <Bundle-Description>Jakarta JSON Processing API ${spec.version}</Bundle-Description>
                         <Specification-Vendor>${vendor.name}</Specification-Vendor>
                         <Implementation-Build-Id>${buildNumber}</Implementation-Build-Id>
-                        <DynamicImport-Package>*</DynamicImport-Package>
+                        <DynamicImport-Package>
+                            !org.eclipse.parsson,
+                            *
+                        </DynamicImport-Package>
```

Comment:
What error is this trying to fix? Why exactly this change helps?

Reply:
> What error is this trying to fix? 

OSGi startup error. The specific error is posted in the issue below:
https://github.com/eclipse-ee4j/jsonp/issues/373#issue-1206326916

> Why exactly this change helps?

the error is raised by jakarta.json-api requiring org.eclipse.parasson as osgi.wiring.package oddly. so, like Import-Package, explicitly excluded org.eclipse.parasson from DynamicImport-Package to avoid the error.

Reply:
I tried to reproduce the error on the glassfish side but had no luck. Do you happen to have a GF binary where one can reproduce the error you see?

I'd like to understand why valid OSGi header is causing troubles. Do you know?

Reply:
Sorry for the late reply.  
* Here is the binary that reproduces the problem:  
  https://drive.google.com/drive/folders/13bFlw3asz3vymi1wiPXuCHhC9I5C04eP?usp=sharing
* This binary was built with the steps described in the following issue, except for jsonp:  
  https://github.com/eclipse-ee4j/glassfish/issues/23782#issuecomment-1100839280

Reply:
The error can be reproduced by accessing the management console after performing the following steps:
```bash
unzip glassfish.zip
cd glassfish7/glassfish/bin
asadmin start-domain
asadmin change-admin-password ##The default admin password is empty.
asadmin enable-secure-admin
asadmin restart-domain
curl -sk https://localhost:4848/
```

Reply:
Thank you for your time! Clearing the local repository resolved this error